README.md

```diff
@@ -0,0 +1,191 @@
+
+- HTML-5
+- CSS3
+- Linters
+- Figma
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[Navigation bar]**
+- **[Header section - Introduction]**
+- **[Work Portfolio section - Project description]**
+- **[About Section - About Myself]**
+- **[Contact-me-Section - Contact-Form]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Added a link to my deployed project.
+
+- [Live Demo Link](https://mariamosman11.github.io/Portflio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- An IDE (Intergrated Development Environment ) such as vsode or atom  
+- A version control system such as git
+
+
+### Setup
+
+1.  Clone the Repo with:
+```
+git clone https://github.com/migo1/portfolio
+```
+2. switch to the cloned directory
+```
+cd portfolio
+```
+
+### Installation
+
+1. Install the project with it's depenedcies
+```
+npm install
+```
+Then open the live server or open the `index.html` file with your browser
+
+### Usage
+
+- this is the portfolio project I'll use to update more changes.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "**/*.{css,scss}" 
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+- [🚀 Live Demo Link Provided Above ⬆️](#live-demo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: (https://github.com/migo1)
+- Twitter: (https://twitter.com/@ert49554392)
+- LinkedIn: (https://www.linkedin.com/in/robert-m-3b3b1116a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added:
+
+- [ ] **[Project: Application deploy]**
+- [ ] **[Project: Evaluate accessibility (pair programming)]**
+- [ ] **[Project: Mobile menu (pair programming)]**
+- [ ] **[Project: Details modal (pair programming)]**
+- [ ] **[Project: Validate contact form (pair programming)]**
+- [ ] **[Project: Preserve data in the browser (pair programming)]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]()
```

Comment:
- [ ] please, like the previous reviewer noted, its good to avoid broken links in your README file. Please kindly a the link to your issue page here. Nice work

![Screenshot (1004)](https://user-images.githubusercontent.com/65298048/234924311-4e3e6be2-f063-434f-9791-05c81f989ab2.png)

Reply:
noted